src/Build/BackEnd/Shared/BuildRequestConfiguration.cs

```diff
@@ -264,33 +254,19 @@ private BuildRequestConfiguration(INodePacketTranslator translator)
         /// Flag indicating whether the configuration is allowed to cache.  This does not mean that the configuration will
         /// actually cache - there are several criteria which must for that.
         /// </summary>
-        public bool IsCacheable
-        {
-            get;
-            set;
-        }
+        public bool IsCacheable { get; set; }
 
         /// <summary>
         /// When reset caches is false we need to only keep around the configurations which are being asked for during the design time build.
         /// Other configurations need to be cleared. If this configuration is marked as ExplicitlyLoadedConfiguration then it should not be cleared when 
         /// Reset Caches is false.
         /// </summary>
-        public bool ExplicitlyLoaded
-        {
-            get;
-            set;
-        }
+        public bool ExplicitlyLoaded { get; set; }
 
         /// <summary>
         /// Flag indicating whether or not the configuration is actually building.
         /// </summary>
-        public bool IsActivelyBuilding
-        {
-            get
-            {
-                return (_activelyBuildingTargets != null) && (_activelyBuildingTargets.Count > 0);
-            }
-        }
+        public bool IsActivelyBuilding => (_activelyBuildingTargets != null) && (_activelyBuildingTargets.Count > 0);
```

Comment:
Further simplification to

```c#
_activelyBuildingTargets?.Count > 0
```

?

Reply:
That reduces to `int? > 0` which I don't think works. You'd need to do something like `(_activelyBuildingTargets?.Count ?? 0) > 0`, but IMHO that's worse than what's there now.

Reply:
`int? {comparison} int` is [well-defined](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/nullable-types/using-nullable-types#operators): it returns `false`. So I still think I want this change.